src/OrleansSQLUtils/Storage/RelationalStorage.cs

```diff
@@ -146,11 +146,7 @@ public Task<IEnumerable<TResult>> ReadAsync<TResult>(string query, Action<IDbCom
                 throw new ArgumentNullException("selector");
```

Comment:
All right, in `SqlStatisticsPublisher` the line `this.grainReferenceConverter = providerRuntime.ServiceProvider.GetRequiredService<IGrainReferenceConverter>();` throws `NotImplementedException`. I don't know why, though, I'm gearing to train here. :)